src/DurableFunctions.CodeGeneration/WebJobs.Extensions.DurableTask.CodeGen.sln

```diff
@@ -0,0 +1,43 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31205.134
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DurableFunctions.CodeGen.Example", "Example\DurableFunctions.CodeGen.Example.csproj", "{0C0D1182-4069-4895-9D5D-6C8A6C4D8E25}"
```

Comment:
Are we ok with breaking the WebJobs.Extensions.DurableTask theme to switch to DurableFunctions.CodeGen?

Reply:
I think so. We can always reframe it if we officially integrate it into Durable Functions, but for now I'm fine with a less official name.